src/components/App/App.js

```diff
@@ -39,11 +41,17 @@ class App extends Component {
         path="/movies/:id/" exact
         render={({ match }) => {
           const selectedMovie = this.props.movies.find(movie => parseInt(match.params.id) === movie.id)
+
           return <MovieDetails
                   {...selectedMovie}
                  />
         }}
       />
+      <Route
+        path="*"
```

Comment:
Just to make sure I understand, is this basically a catch-all for any paths that don't fit other routes?

Reply:
Yep! And actually, this would still work without including the path="*" so we can take that out for simplicity if we want. The Switch tells it to go through each Route until it hits one that matches.

Reply:
Oooh. Gotcha. Thanks!